### PR TITLE
Post Excerpt block: Added icon and description to Post Excerpt block

### PR DESCRIPTION
**Pull request by mapk**
## Description
Added icon and description to Post Excerpt block.

## How has this been tested?
Tested locally.

## Screenshots 

Screen Shot 2020-07-24 at 2 14 39 PM

## Types of changes
Non-breaking changes.

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

**Comment by mapk** [comment]
I updated the excerpt description to read, "Display a post's excerpt."

File: packages/icons/src/library/post-excerpt.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ */
+import { Path, SVG } from '@wordpress/primitives';
+
+const postExcerpt = (
+	<SVG fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+		<Path
+			fill-rule="evenodd"
+			clip-rule="evenodd"
+			d="M12.75 9.333c0 .521-.102.977-.327 1.354-.23.386-.555.628-.893.774-.545.234-1.183.227-1.544.222l-.12-.001v-1.5h.123c.414.001.715.002.948-.099a.395.395 0 00.199-.166c.05-.083.114-.253.114-.584V7.2H8.8V4h3.95v5.333zM7.95 9.333c0 .521-.102.977-.327 1.354-.23.386-.555.628-.893.774-.545.234-1.183.227-1.544.222l-.12-.001v-1.5h.123c.414.001.715.002.948-.099a.394.394 0 00.198-.166c.05-.083.115-.253.115-.584V7.2H4V4h3.95v5.333zM13 20H4v-1.5h9V20zM20 16H4v-1.5h16V16z"
+			fill="#1E1E1E"
```
Comment (MichaelArestad) [inline packages/icons/src/library/post-excerpt.js:12]: I would remove this fill.

File: packages/icons/src/library/post-excerpt.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { Path, SVG } from '@wordpress/primitives';
+
+const postExcerpt = (
+	<SVG fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```
Comment (MichaelArestad) [inline packages/icons/src/library/post-excerpt.js:7]: Remove this fill as well.  (The fills are defined in the CSS so no need here)

File: packages/icons/src/library/post-excerpt.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+		<Path
+			fill-rule="evenodd"
+			clip-rule="evenodd"
```
Comment (MichaelArestad) [inline packages/icons/src/library/post-excerpt.js:10]: I'm not sure about these clip and fill rules. I don't really know if they are needed.

**Review by MichaelArestad — CHANGES_REQUESTED** [review]
This looks good. I think the description is short and sweet. I do have some minor things for you to look into on the icon and then it's good to go. Nice work!

**Review by MichaelArestad — APPROVED** [review]

**Pull request merged**